test/run.sh

```diff
@@ -101,6 +105,7 @@ else
     # shellcheck disable=SC2086
     "${RF_BINARY}" \
         --randomize all \
+        --prerunmodifier resources/SkipTests.py:${SKIP_TESTS:-} \
```

Comment:
Does this work if you do not run from the `microshift/tests` current directory?
We should probably specify a full path to the script, i.e. `${SCRIPTDIR}/resources/SkipTests.py`

Reply:
It does work, new commit pushed to set the full path.